storage/store_metrics.go

```diff
@@ -26,30 +25,43 @@ import (
 )
 
 var (
-	metaReplicaCount                 = metric.Metadata{Name: "replicas"}
-	metaReservedReplicaCount         = metric.Metadata{Name: "replicas.reserved"}
-	metaLeaderRangeCount             = metric.Metadata{Name: "ranges.leader"}
-	metaReplicatedRangeCount         = metric.Metadata{Name: "ranges.replicated"}
-	metaReplicationPendingRangeCount = metric.Metadata{Name: "ranges.replication-pending"}
-	metaAvailableRangeCount          = metric.Metadata{Name: "ranges.available"}
-	metaLeaseRequestSuccessCount     = metric.Metadata{Name: "leases.success"}
-	metaLeaseRequestErrorCount       = metric.Metadata{Name: "leases.error"}
-	metaLiveBytes                    = metric.Metadata{Name: "livebytes"}
-	metaKeyBytes                     = metric.Metadata{Name: "keybytes"}
-	metaValBytes                     = metric.Metadata{Name: "valbytes"}
-	metaIntentBytes                  = metric.Metadata{Name: "intentbytes"}
-	metaLiveCount                    = metric.Metadata{Name: "livecount"}
-	metaKeyCount                     = metric.Metadata{Name: "keycount"}
-	metaValCount                     = metric.Metadata{Name: "valcount"}
-	metaIntentCount                  = metric.Metadata{Name: "intentcount"}
-	metaIntentAge                    = metric.Metadata{Name: "intentage"}
-	metaGcBytesAge                   = metric.Metadata{Name: "gcbytesage"}
-	metaLastUpdateNanos              = metric.Metadata{Name: "lastupdatenanos"}
-	metaCapacity                     = metric.Metadata{Name: "capacity"}
-	metaAvailable                    = metric.Metadata{Name: "capacity.available"}
-	metaReserved                     = metric.Metadata{Name: "capacity.reserved"}
-	metaSysBytes                     = metric.Metadata{Name: "sysbytes"}
-	metaSysCount                     = metric.Metadata{Name: "syscount"}
+	// Replica metrics.
```

Comment:
since you're renaming a bunch of metrics, you should go ahead and fill in the help strings